libs/agno/agno/memory/v2/memory.py

```diff
@@ -80,18 +81,10 @@ class Memory:
     # Manager to manage memories
     memory_manager: Optional[MemoryManager] = None
 
-    # Session summaries per session per user
-    summaries: Optional[Dict[str, Dict[str, SessionSummary]]] = None
-    # Summarizer to generate session summaries
-    summary_manager: Optional[SessionSummarizer] = None
+    db: Optional[BaseDb] = None
 
-    db: Optional[MemoryDb] = None
-
-    # runs per session
-    runs: Optional[Dict[str, List[Union[RunResponse, TeamRunResponse]]]] = None
-
-    # Team context per session
-    team_context: Optional[Dict[str, TeamContext]] = None
+    # # Team context per session
+    # team_context: Optional[Dict[str, TeamContext]] = None
```

Comment:
Yeah lets get back to this, I think we should get rid of it and ONLY work with team_session_state for shared context